cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerDeployClientConfigProvider.java

```diff
@@ -258,4 +253,17 @@ private void enhanceRequestParams(Map<String, List<String>> headers, List<Pair>
     ExecutorService createExecutor() {
         return Executors.newSingleThreadExecutor();
     }
+
+    @VisibleForTesting
```

Comment:
a good sign this class already needs refactoring

Reply:
agreed, but i think i won't do that now:)

Reply:
the whole code should be removed anyway if the CM command will become async